includes/tracks/class-wc-tracks-client.php

```diff
@@ -92,10 +92,49 @@ public static function build_timestamp() {
 		return number_format( $ts, 0, '', '' );
 	}
 
+	/**
+	 * Get a user's identity to send to Tracks. If Jetpack exists, default to its implementation.
+	 *
+	 * @param int $user_id User id.
+	 * @return array Identity properties.
+	 */
+	public static function get_identity( $user_id ) {
+		if ( class_exists( 'Jetpack' ) ) {
+
+			include_once ABSPATH . 'wp-content/plugins/jetpack/_inc/lib/tracks/client.php';
```

Comment:
This path may change pending in the WP installation, we should use `WP_PLUGIN_DIR` instead.

```php
include_once trailingslashit( WP_PLUGIN_DIR ) . 'jetpack/_inc/lib/tracks/client.php';
```